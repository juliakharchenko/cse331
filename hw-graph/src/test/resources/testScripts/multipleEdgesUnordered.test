# Tests if edges are printed in alphabetical order

# Create a graph
CreateGraph multiEdgeGraph

# Add nodes
AddNode multiEdgeGraph a
AddNode multiEdgeGraph b

# Add edges
AddEdge multiEdgeGraph a b AB2
AddEdge multiEdgeGraph a b AB1
AddEdge multiEdgeGraph b a BA2
AddEdge multiEdgeGraph b a BA1

# Prints the nodes in the graph and the outgoing edges from a and b
ListNodes multiEdgeGraph
ListChildren multiEdgeGraph a
ListChildren multiEdgeGraph b