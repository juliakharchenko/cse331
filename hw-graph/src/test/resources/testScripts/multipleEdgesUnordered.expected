# Tests if edges are printed in alphabetical order

# Create a graph
created graph multiEdgeGraph

# Add nodes
added node a to multiEdgeGraph
added node b to multiEdgeGraph

# Add edges
added edge AB2 from a to b in multiEdgeGraph
added edge AB1 from a to b in multiEdgeGraph
added edge BA2 from b to a in multiEdgeGraph
added edge BA1 from b to a in multiEdgeGraph

# Prints the nodes in the graph and the outgoing edges from a and b
multiEdgeGraph contains: a b
the children of a in multiEdgeGraph are: b(AB1) b(AB2)
the children of b in multiEdgeGraph are: a(BA1) a(BA2)